templates/base/packages/nextjs/app/layout.tsx.template.mjs

```diff
@@ -1,3 +1,8 @@
+import { withDefaults } from "../../../../utils.js";
+
+const contents = ({ globalCssImports }) => {
```

Comment:
Umm `globalCssImports` seems too specific maybe change to something generic? Like "imports" or something else

Reply:
Thanks, updated [8e028ec](https://github.com/scaffold-eth/create-eth/pull/74/commits/8e028ec65178020965f23b6cac159524ee132363)